django_airavata/apps/admin/static/django_airavata_admin/src/components/admin/group_resource_preferences/GroupComputeResourcePreference.vue

```diff
@@ -97,12 +126,16 @@
       </template>
     </list-layout>
     <div class="fixed-footer">
-      <b-button variant="primary" @click="saveGroupResourceProfile"
+      <b-button 
+      variant="primary" 
+      disabled="!data.userHasWriteAccess"
```

Comment:
this should be a binding, like `:disabled="!data.userHasWriteAccess"`. As it is, the expression isn't being evaluated and is taken as a literal string.